spec/models/report_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  describe '#editable' do
+    let(:user) { User.create!(**params) }
+    let(:params) do
+      {
+        name: 'alice',
+        email: 'alice@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      }
+    end
+    let(:another_user) { User.create!(**another_params) }
+    let(:another_params) do
+      {
+        name: 'bob',
+        email: 'bob@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      }
+    end
+    let(:report) { Report.create!(**report_params) }
+    let(:report_params) do
+      {
+        user_id: user.id,
+        content: 'content',
+        title: 'title'
+      }
+    end
+    context 'user have report' do
```

Comment:
```suggestion
    context 'user has report' do
```

英文法的にはhasですね。
ただ、頑張って英語にしなくても日本語で書いてもいいですよ（実務では開発チームごとの方針に合わせる必要がありますが）